bin/oneoff/upload_non_en_videos.rb

```diff
@@ -0,0 +1,126 @@
+# This script is used to batch upload non-EN videos, with the assumption that
+# we will frequently get these in batches. They will be uploaded to YouTube
+# and S3 as well as added to videos.csv.
+
+require 'google/apis'
+require 'google/apis/youtube_v3'
+require 'google/api_client/client_secrets.rb'
+require 'googleauth'
+require 'googleauth/stores/file_token_store'
+require 'csv'
+require 'fileutils'
+gem 'google-api-client', '>0.7'
```

Comment:
Worth adding this to the Gemfile?

Reply:
I think it already is. I got a lot of this code from Google documentation. I've removed it.